kodev

```diff
@@ -110,9 +110,9 @@ function kodev-fetch-thirdparty() {
 
 SUPPORTED_TARGETS="
     kindle          For kindle with touch support
-    kindle5         For kindle models >= kindle5 < paper white 2
+    kindle5         For kindle models >= kindle4 < paper white 2
```

Comment:
Since the kindle5 target is used for nothing except compiler executable selection maybe it'd be clearer to just rename it to kindle4?

Btw, don't forget the README in base. :-)

Reply:
> Since the kindle5 target is used for nothing except compiler executable selection maybe it'd be clearer to just rename it to kindle4?

The toolchain in base is called arm-kindle5... , so it can be confusing if I change it here

Reply:
I don't think the name of the TC matters much for the name of the target. In fact I think base/Makefile.defs is the perfect place to abstract such confusion away, if any.

Even more correctly, we should just change https://github.com/koreader/koxtoolchain/blob/47acb959b6169b5d1e546814d1747a8f88484b1b/configs/ct-ng-kindle5-config#L188

Reply:
Even more correctly, we should just remove kindle5 and kindlepw entirely since kindle is already doing the job. But I just wanted to make a quick correction, and it doesn't matter much since it's already working well.

Reply:
I'm not sure if that follows. I imagine the kindle5/pw2 targets are more highly optimized. According to https://software.intel.com/en-us/blogs/2012/09/26/gcc-x86-performance-hints (admittedly not directly relevant for ARM) the performance difference can be as high as 43% between generic and optimized. Anyway, you'd have to ask @NiLuJe about the specifics.

Reply:
By which I mean, the 'MR' convention is to assume K5 means anything w/ a touchscreen (i.e., anything since the Kindle Touch).

That only leaves the K4 in an odd place; because it's basically the K5 hardware, but inside the shell of a K3, and with a bastardized version of the firmware that tries to fit K5 concepts into the K3 FW.

Reply:
So, yeah, naming it K5 made sense for me at the time, because if matched the split in basically everything we'd done at that point because of the switch to a touchscreen and an updated firmware (upstart, the new UI, ...).

But I definitely get @Frenzie's reasoning, in the context of KOReader, naming the whole thing kindle4 might indeed make more sense.